Examples/ExamplesApp/Examples.swift

```diff
@@ -15,57 +15,70 @@
 import SwiftUI
 
 struct Examples: View {
-    /// The categories to display.
-    let categories = makeCategories()
-    
-    /// The category selected by the user.
-    @State private var selectedCategory: Category?
-    /// The example selected by the user.
-    @State private var selectedExample: Example?
+    /// The menu items to display.
+    let menuItems = makeMenuItems()
     
     var body: some View {
         NavigationSplitView {
             NavigationStack {
-                List(categories, id: \.name, selection: $selectedCategory) { category in
-                    NavigationLink(category.name) {
-                        List(category.examples, id: \.name, selection: $selectedExample) { example in
-                            Text(example.name)
-                                .tag(example)
+                List(menuItems, id: \.name) { item in
+                    if let category = item as? Category {
+                        NavigationLink(category.name) {
+                            List(category.examples, id: \.name) { example in
+                                makeExampleLink(example)
+                            }
+                            .listStyle(.sidebar)
+                            .navigationTitle(category.name)
+                            .navigationBarTitleDisplayMode(.inline)
                         }
-                        .listStyle(.sidebar)
-                        .navigationTitle(category.name)
-                        .navigationBarTitleDisplayMode(.inline)
+                        .isDetailLink(false)
+                    } else if let example = item as? Example {
+                        makeExampleLink(example)
                     }
-                    .isDetailLink(false)
                 }
                 .navigationTitle("Toolkit Examples")
             }
         } detail: {
-            NavigationStack {
-                if let example = selectedExample {
-                    example.view
-                        .navigationTitle(example.name)
-                        .navigationBarTitleDisplayMode(.inline)
-                } else if selectedCategory != nil {
-                    Text("Select an example")
-                } else {
-                    Text("Select a category")
-                }
-            }
+            Text("Select an example")
```

Comment:
This text does not make as much sense now that the sidebar has both examples and categories.

Reply:
With the DisclosureGroup implemented, I think this makes more sense to keep as is now. Thoughts?

Reply:
Yep, sounds good.